frontend/src/data/nsf313_30/categories.ts

```diff
@@ -0,0 +1,18 @@
+export const Categories = {
+    'race': new Set<string>([
+        'Black'
+    ]),
+    'collegeOrUniversityLevel': new Set<string>([
+        '4year',
+        '2year',
+    ]),
+    'collegeOrGraduateSchoolEnrollment': new Set<string>([
+        'EnrolledInCollegeOrGraduateSchool', 
```

Comment:
Can we can remove this summary statistic dimension?

Reply:
done